app/src/main/java/ar/com/wolox/android/example/ui/newsdetail/NewsDetailFragment.kt

```diff
@@ -0,0 +1,69 @@
+package ar.com.wolox.android.example.ui.newsdetail
+
+import android.graphics.Typeface
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.FragmentNewsDetailBinding
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.android.example.ui.home.HomeActivity
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import org.ocpsoft.prettytime.PrettyTime
+
+class NewsDetailFragment constructor(private val news: News) : WolmoFragment<FragmentNewsDetailBinding, NewsDetailPresenter>(), NewsDetailView {
+    private var newsDetail = news
```

Comment:
I think the news should not be saved in the fragment but in the presenter. Try calling a `presenter.onInit(news)` method in the fragment's init.